### PR TITLE
surya/core 5634/gregor more chat info

### PR DESCRIPTION
**Pull request by modalduality**

### CODE REVIEW

File: go/chat/server.go [hunk 1]
```diff
@@ -2272,10 +2272,11 @@ func (h *Server) JoinConversationLocal(ctx context.Context, arg chat1.JoinConver
...
 	teamConvs, err := h.remoteClient().GetTLFConversations(ctx, chat1.GetTLFConversationsArg{
-		TlfID:            nameInfo.ID,
-		MembersType:      chat1.ConversationMembersType_TEAM,
-		TopicType:        arg.TopicType,
-		SummarizeMaxMsgs: false, // tough call here, depends on if we are in most of convos on the team
+		TlfID:                nameInfo.ID,
+		MembersType:          chat1.ConversationMembersType_TEAM,
+		TopicType:            arg.TopicType,
+		SummarizeMaxMsgs:     false, // tough call here, depends on if we are in most of convos on the team
+		IncludeAuxiliaryInfo: false,
```
Comment (modalduality) [inline go/chat/server.go]: not sure where this should be prefilled in as true or false on client side

File: go/engine/favorite_test.go [hunk 2]
```diff
@@ -1,261 +0,0 @@
-// Copyright 2015 Keybase, Inc. All rights reserved. Use of
```
Comment (patrickxb) [inline go/engine/favorite_test.go]: did you mean to delete this file as part of this PR?

File: go/chat/server.go (hunk 1 above)
Comment (patrickxb) [inline go/chat/server.go]: I think the rpc client should specify if they want it.  I'd make it an arg in helpers.go:GetTLFConversations(), and add it it as an arg to GetConversationsLocal() in the chat protocol.  Then just leave it false everywhere else.

File: go/chat/helper.go [hunk 3]
```diff
@@ -268,10 +268,11 @@ func GetTLFConversations(ctx context.Context, g *globals.Context, debugger utils
...
 	tlfRes, err := ri().GetTLFConversations(ctx, chat1.GetTLFConversationsArg{
-		TlfID:            tlfID,
-		TopicType:        topicType,
-		MembersType:      membersType,
-		SummarizeMaxMsgs: false,
+		TlfID:                tlfID,
+		TopicType:            topicType,
+		MembersType:          membersType,
+		SummarizeMaxMsgs:     false,
+		IncludeAuxiliaryInfo: true,
```
Comment (patrickxb) [inline go/chat/helper.go]: See my other comment.  Perhaps this should be controlled by argument to this function.

**Comment by keybase-ci-visdiff** [comment]
The commits 71117ddd9d0b...70264f977fa0 introduce visual changes on linux. <details><summary>:mag_right: 1 changed</summary>

 * Changed: **Markdown-emoji** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1501514792719-71117ddd9d0b-70264f977fa0-linux/Markdown-emoji-71117ddd9d0b.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1501514792719-71117ddd9d0b-70264f977fa0-linux/Markdown-emoji-70264f977fa0.png)  
   Markdown-emoji comparison
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits 71117ddd9d0b...70264f977fa0 introduce visual changes on linux. <details><summary>:mag_right: 1 changed</summary>

 * Changed: **Markdown-emoji** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1501524246426-71117ddd9d0b-70264f977fa0-linux/Markdown-emoji-71117ddd9d0b.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1501524246426-71117ddd9d0b-70264f977fa0-linux/Markdown-emoji-70264f977fa0.png)  
   Markdown-emoji comparison
</summary>

**Comment by modalduality** [comment]
@mmaxim

**Comment by mmaxim** [comment]
Looks good pending @patrickxb

**Pull request merged**